### PR TITLE
add section to readme about connecting to notary server over tor 0:)

### PR DESCRIPTION
**Pull request by jfrazelle**
ping @NathanMcCauley

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,3 +141,57 @@ The configuration file must be a json file with the following format:
...
+    jess/privoxy
+
+# check the socks proxy/http proxy work
+$ curl --socks http://localhost:9050  -L https://check.torproject.org/api/ip
+# {"IsTor":true,"IP":"exit.node.ip.addr"}
+$ curl -x http://localhost:8118  -L https://check.torproject.org/api/ip
+# {"IsTor":true,"IP":"exit.node.ip.addr"}
+```
+
+Then it is as simple as exporting the `http_proxy` enviornment variable:
```
Comment (thaJeztah) [inline README.md]: s/enviornment/environment/ :smile:

Comment (jfrazelle) [inline README.md]: that word and privileged, gets me errytime thanks :)

**Comment by diogomonica** [comment]
This is great material for a blogpost, but I don't think it should be in our README.

:-1:

**Comment by jfrazelle** [comment]
what no!

**Comment by jfrazelle** [comment]
what about a different page

**Comment by jfrazelle** [comment]
NATHANNNNN

**Comment by jfrazelle** [comment]
(im kidding its your repo do what you want)

**Comment by jfrazelle** [comment]
i _will_ remember this though the next time you want something merged
elsewhere :p

**Comment by diogomonica** [comment]
:fearful:

**Pull request closed without merging**